tools/make/kube.mk

```diff
@@ -96,6 +96,25 @@ kube-demo-undeploy: ## Uninstall the Kubernetes resources installed from the `ma
 .PHONY: conformance
 conformance: create-cluster kube-install-image kube-deploy run-conformance delete-cluster ## Create a kind cluster, deploy EG into it, run Gateway API conformance, and clean up.
 
+.PHONY: e2e
+e2e: create-cluster kube-install-image kube-deploy install-ratelimit run-e2e delete-cluster
+
+.PHONY: install-ratelimit
+install-ratelimit:
+	@$(LOG_TARGET)
+	kubectl apply -f examples/redis/redis.yaml
```

Comment:
only nit here would be to split up `redis` install with patching/updating the `EnvoyGateway` Configmap, but it can be improved later